ONSdigital/dp-frontend-articles-controller
# Task/empty ons design system route

### What

This reintroduces the `/` handler, now rendering through the ONS Design System.

The template is only a placeholder, and the text 'Release date' is temporary and only serves to demonstrate locale switching working in the template. It will be superceded by the actual page layout in a subsequent PR.

- Upgrade dp-api-clients-go/v2
- Upgrade log.go
- Upgrade Go Convey
- Enable ONS Design System on `/` and upgrade dp-renderer, dp-design-system
- Enable localisation toggle in header (support `lang` cookie set to `en` or `cy`)
- Reinstate /bulletin route with an empty ONS Design System template

<img alt="Screenshot 2022-05-13 at 16 01 48" src="https://user-images.githubusercontent.com/912770/168312111-2826b9d3-61da-4843-a2e2-11193e967107.png">

![Screenshot 2022-05-13 at 16 01 20](https://user-images.githubusercontent.com/912770/168312121-b5cbee56-7a3a-4719-8a98-e0ecc20a1c81.png)


### How to review

- In a separate shell run dp-design-system with `make debug`
- Run this frontend with `make debug`
- Visit `http://localhost:26500/economy/grossdomesticproductgdp/bulletins/gdpmonthlyestimateuk/august2019`
- Confirm page renders the empty template as shown in screenshots

- Stop the dp-design-system server
- In a separate shell run sixteens with `./run.sh`
- Visit `http://localhost:26500/sixteens/economy/grossdomesticproductgdp/bulletins/gdpmonthlyestimateuk/august2019`
- Confirm page renders a full bulletin

### Who can review

Go developers
